status/src/main/resources/rp-tooling.conf

```diff
@@ -0,0 +1,9 @@
+akka {
+  extensions += "com.lightbend.rp.status.ApplicationStatus"
+
+  management {
+    http {
+      route-providers += "com.lightbend.rp.status.ApplicationStatus"
```

Comment:
was this not possible in reference.conf?

Reply:
I checked impl, should have worked

Reply:
I didn't try reference.conf -- any settings for other libraries in this project we place in rp-tooling.conf. The SBT plugin ensures these settings are overlaid above reference.conf but below application.conf.